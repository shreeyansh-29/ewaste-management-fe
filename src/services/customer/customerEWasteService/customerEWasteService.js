import {CUSTOMER_VIEW_DRIVES} from "../../../container/constant/constants";
import api from "../../../core/utilities/httpProvider";

export const customerEWasteService = () => {
  return api.get(CUSTOMER_VIEW_DRIVES);
};
