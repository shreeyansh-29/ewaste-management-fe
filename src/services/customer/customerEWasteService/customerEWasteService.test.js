/* eslint-disable no-undef */

import {customerEWasteService} from "./customerEWasteService";
import React from "react";
import Enzyme, {shallow} from "enzyme";
import toJson from "enzyme-to-json";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";

Enzyme.configure({adapter: new Adapter()});

it("testing customerEWasteService", () => {
  const Service = customerEWasteService;
  const wrapper = shallow(<Service />);
  expect.assertions(1);
  expect(toJson(wrapper)).toMatchSnapshot();
});
