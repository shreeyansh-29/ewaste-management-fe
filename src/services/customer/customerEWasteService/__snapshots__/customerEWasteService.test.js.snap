// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing customerEWasteService 1`] = `<Component />`;
